src/applications/simple-forms/form-upload/config/constants.js

```diff
@@ -9,6 +11,11 @@ export const PrimaryActionLink = ({ href = '/', children, onClick = null }) => (
     </div>
   </div>
 );
+PrimaryActionLink.propTypes = {
+  children: PropTypes.element.isRequired,
+  href: PropTypes.string.isRequired,
+  onClick: PropTypes.func.isRequired,
+};
```

Comment:
Adding this because of a linter warning.

Reply:
`PropTypes.string` and `PropTypes.func` should suffice.